### DISCUSSION TITLE
Account Deletion - TShock.co

### DISCUSSION THREAD

**Discussion by JordanPlayz158** [Pryaxis/TShock#2113] [General]
I was wondering if any staff members of TShock.co (which was moved to this github) could delete my account from the old website?

**Comment by hakusaro** [comment]
Yeah, but you have to send a forum PM to nicatronTg to request that.
GitHub <notifications@github.com> wrote:
“I was wondering if any staff members of TShock.co (which was moved to this github) could delete my account from the old website?”

**Reply by JordanPlayz158** [reply]
Thank you!

**Comment by hakusaro** [comment]
No problem!
GitHub <notifications@github.com> wrote:
“Thank you!”
